src/Integration/ExceptionListenerIntegration.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Integration;
+
+use Sentry\ErrorHandler;
+use Sentry\State\Hub;
+
+/**
+ * This integration hooks into the global error handlers and emits events to
+ * Sentry.
+ */
+final class ExceptionListenerIntegration implements IntegrationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function setupOnce(): void
+    {
+        ErrorHandler::addExceptionListener(function (\Throwable $throwable): void {
```

Comment:
Since this function doesn't need access to `$this` it can be `static`

Reply:
That would break the interface, and limit the extension possibilities for the end user.

Reply:
Sorry I was not clear enough, I mean the lambda function. You can declare it as `static function (\Throwable $throwable): void` if you don't need to access `$this` inside it: http://php.net/manual/en/functions.anonymous.php#functions.anonymous-functions.static